### ISSUE TITLE
Error code list which need to be updated to new format

### ISSUE THREAD

**Issue by GuillaumeGomez** [rust-lang/rust#35233]
If you want to work on one (or more) of the issues, just comment on it to say you're taking it!

You can read @jonathandturner's [post about how to make the fixes](http://www.jonathanturner.org/2016/08/helping-out-with-rust-errors.html).  If you have any questions, ask us here or on #rust-internals on irc.mozilla.org.

Also, if you finish an issue and it has a "Bonus" part that you're not able to complete, just go ahead and close the issue and open a new issue for just the Bonus part.  That way, someone else can jump in and work on the bonus part. You can cc me on the new issue and we can add it to the list above.

Here is the list of error codes:
- [x] [E0001](https://github.com/rust-lang/rust/issues/35190) 🔒 @saml 
- [x] [E0002](https://github.com/rust-lang/rust/issues/35495) 🔒 @GuillaumeGomez
- [x] [E0004](https://github.com/rust-lang/rust/issues/35191) 🔒 @TheZoq2
- [x] [E0005](https://github.com/rust-lang/rust/issues/35192) 🔒 @pythoneer
- [x] [E0007](https://github.com/rust-lang/rust/issues/35496) 🔒 @srdja
- [x] [E0008](https://github.com/rust-lang/rust/issues/35496) 🔒 @srdja
- [x] [E0009](https://github.com/rust-lang/rust/issues/35193) 🔒 @DevShep
- [x] [E0010](https://github.com/rust-lang/rust/issues/35194) 🔒 @pcn
- [x] [E0017](https://github.com/rust-lang/rust/issues/35195) 🔒 @pcn
- [x] [E0023](https://github.com/rust-lang/rust/issues/35196) 🔒 @pcn
- [x] [E0025](https://github.com/rust-lang/rust/issues/35198)
- [x] [E0026](https://github.com/rust-lang/rust/issues/35497) 🔒 @razielgn
- [x] [E0027](https://github.com/rust-lang/rust/issues/35200) 🔒 @silenuss
- [x] [E0029](https://github.com/rust-lang/rust/issues/35201) 🔒 @silenuss
- [x] [E0030](https://github.com/rust-lang/rust/issues/35204) 🔒 @silenuss
- [x] [E0033](https://github.com/rust-lang/rust/issues/35498) 🔒 @munyari
- [x] [E0034](https://github.com/rust-lang/rust/issues/35205) 🔒 @neilzxu
- [x] [E0035](https://github.com/rust-lang/rust/issues/35206) 🔒 @neilzxu
- [x] [E0036](https://github.com/rust-lang/rust/issues/35207) 🔒 @neilzxu
- [x] [E0038](https://github.com/rust-lang/rust/issues/35500) 🔒 @munyari
- [x] [E0040](https://github.com/rust-lang/rust/issues/35208) 🔒 @shri3k
- [x] [E0045](https://github.com/rust-lang/rust/issues/35501) 🔒 @mikhail-m1
- [x] [E0046](https://github.com/rust-lang/rust/issues/35209) 🔒 @shri3k 
- [x] [E0049](https://github.com/rust-lang/rust/issues/35210) 🔒 @GuillaumeGomez
- [x] [E0050](https://github.com/rust-lang/rust/issues/35211) 🔒 @GuillaumeGomez
- [x] [E0054](https://github.com/rust-lang/rust/issues/35791) 🔒 @knight42
- [x] [E0055](https://github.com/rust-lang/rust/issues/35213) 🔒 @nickmass 
- [x] [E0057](https://github.com/rust-lang/rust/issues/35214)  
- [x] [E0060](https://github.com/rust-lang/rust/issues/35215) 🔒 @birryree
- [x] [E0061](https://github.com/rust-lang/rust/issues/35216) 🔒 @jaredwy
- [x] [E0062](https://github.com/rust-lang/rust/issues/35217) 🔒 @trixnz
- [x] [E0063](https://github.com/rust-lang/rust/issues/35218) 🔒 @jaredwy
- [x] [E0067](https://github.com/rust-lang/rust/issues/35502) 🔒 @clementmiao
- [x] [E0069](https://github.com/rust-lang/rust/issues/35219) 🔒 @jaredwy
- [x] [E0070](https://github.com/rust-lang/rust/issues/35503) 🔒 @clementmiao
- [x] [E0071](https://github.com/rust-lang/rust/issues/35220) 🔒 @razielgn
- [x] [E0072](https://github.com/rust-lang/rust/issues/35506) 🔒 @circuitfox
- [x] [E0076](https://github.com/rust-lang/rust/issues/35221) 🔒 @abhiQmar
- [x] [E0079](https://github.com/rust-lang/rust/issues/35222) 🔒 @yossi-k
- [x] [E0080](https://github.com/rust-lang/rust/issues/35223) 🔒 @shantanuraj
- [x] [E0081](https://github.com/rust-lang/rust/issues/35224) 🔒 @poveda-ruiz
- [x] [E0084](https://github.com/rust-lang/rust/issues/35792) 🔒 @pliniker
- [x] [E0087](https://github.com/rust-lang/rust/issues/35225) 🔒 @poveda-ruiz
- [x] [E0088](https://github.com/rust-lang/rust/issues/35226)
- [x] [E0089](https://github.com/rust-lang/rust/issues/35227) 🔒 @zjhmale
- [x] [E0091](https://github.com/rust-lang/rust/issues/35228) 🔒 @Vassah
- [x] [E0092](https://github.com/rust-lang/rust/issues/35229) 🔒 @Vassah
- [x] [E0093](https://github.com/rust-lang/rust/issues/35230) 🔒 @DarkEld3r 
- [x] [E0094](https://github.com/rust-lang/rust/issues/35231) 🔒 @theypsilon
- [x] [E0101](https://github.com/rust-lang/rust/issues/35243) 🔒 @franleplant
- [x] [E0102](https://github.com/rust-lang/rust/issues/35244) 🔒 @franleplant
- [x] [E0106](https://github.com/rust-lang/rust/issues/35245) 🔒 @Tiwalun
- [x] [E0107](https://github.com/rust-lang/rust/issues/35246) 🔒 @oijazsh
- [x] [E0109](https://github.com/rust-lang/rust/issues/35247) 🔒 @circuitfox
- [x] [E0110](https://github.com/rust-lang/rust/issues/35248) 🔒 @circuitfox
- [x] [E0116](https://github.com/rust-lang/rust/issues/35249) 🔒 @terrynsun
- [x] [E0117](https://github.com/rust-lang/rust/issues/35250) 🔒 @Detegr
- [x] [E0118](https://github.com/rust-lang/rust/issues/35251) 🔒 @Detegr
- [x] [E0119](https://github.com/rust-lang/rust/issues/35252) 🔒 @circuitfox
- [x] [E0120](https://github.com/rust-lang/rust/issues/35253) 🔒 @Keats
- [x] [E0121](https://github.com/rust-lang/rust/issues/35254) 🔒 @intrepion
- [x] [E0124](https://github.com/rust-lang/rust/issues/35255) 🔒 @sciyoshi
- [x] [E0128](https://github.com/rust-lang/rust/issues/35508) 🔒 @circuitfox
- [x] [E0130](https://github.com/rust-lang/rust/issues/35256) 🔒 @trixnz
- [x] [E0131](https://github.com/rust-lang/rust/issues/35257) 🔒 @Limeth
- [x] [E0132](https://github.com/rust-lang/rust/issues/35258) 🔒 @GuillaumeGomez
- [x] [E0133](https://github.com/rust-lang/rust/issues/35509) 🔒 @wdv4758h
- [x] [E0137](https://github.com/rust-lang/rust/issues/35265) 🔒 @Keats 
- [x] [E0138](https://github.com/rust-lang/rust/issues/35510) 🔒 @wdv4758h
- [x] [E0162](https://github.com/rust-lang/rust/issues/35268) 🔒 @garekkream 
- [x] [E0164](https://github.com/rust-lang/rust/issues/35269) 🔒 @gavinb
- [x] [E0165](https://github.com/rust-lang/rust/issues/35270) 🔒 @gavinb
- [x] [E0166](https://github.com/rust-lang/rust/issues/35271) 🔒 @Roybie
- [x] [E0172](https://github.com/rust-lang/rust/issues/35272) 🔒 @Roybie
- [x] [E0178](https://github.com/rust-lang/rust/issues/35273) 🔒 @medzin
- [x] [E0184](https://github.com/rust-lang/rust/issues/35275) 🔒 @gavinb
- [x] [E0185](https://github.com/rust-lang/rust/issues/35276) 🔒 @yossi-k 
- [x] E0186 🔒 @yossi-k 
- [x] [E0191](https://github.com/rust-lang/rust/issues/35278) 🔒 @munyari 
- [x] [E0194](https://github.com/rust-lang/rust/issues/35280) 🔒 @birryree
- [x] [E0195](https://github.com/rust-lang/rust/issues/35511) 🔒 @creativcoder 
- [x] [E0201](https://github.com/rust-lang/rust/issues/35304) 🔒 @mikhail-m1
- [x] [E0204](https://github.com/rust-lang/rust/issues/35381) 🔒 @munyari
- [x] [E0205](https://github.com/rust-lang/rust/issues/35382) 🔒 @munyari
- [x] [E0206](https://github.com/rust-lang/rust/issues/35301) 🔒 @KiChjang
- [x] [E0207](https://github.com/rust-lang/rust/issues/35302) 🔒 @shantanuraj
- [x] [E0214](https://github.com/rust-lang/rust/issues/35383) 🔒 @munyari
- [x] [E0220](https://github.com/rust-lang/rust/issues/35385) 🔒 @chamoysvoice
- [x] [E0221](https://github.com/rust-lang/rust/issues/35386) 🔒 @crypto-universe
- [x] [E0223](https://github.com/rust-lang/rust/issues/35387) 🔒 @KiChjang
- [x] [E0225](https://github.com/rust-lang/rust/issues/35388) 🔒 @sciyoshi 
- [x] [E0229](https://github.com/rust-lang/rust/issues/35305) 🔒 @mrabault
- [x] [E0232](https://github.com/rust-lang/rust/issues/35795) 🔒 @mlayne
- [x] [E0243](https://github.com/rust-lang/rust/issues/35389) 🔒 @Keats
- [x] [E0244](https://github.com/rust-lang/rust/issues/35390) 🔒 @Keats
- [x] [E0248](https://github.com/rust-lang/rust/issues/35391) 🔒 @shyaamsundhar
- [x] [E0252](https://github.com/rust-lang/rust/issues/35306) 🔒 @medzin
- [x] [E0253](https://github.com/rust-lang/rust/issues/35512) 🔒 @lukehinds
- [x] [E0254](https://github.com/rust-lang/rust/issues/35513) 🔒 @lukehinds
- [x] [E0259](https://github.com/rust-lang/rust/issues/35514) 🔒 @EugeneGonzalez
- [x] [E0260](https://github.com/rust-lang/rust/issues/35515) 🔒 @0xmohit
- [x] [E0261](https://github.com/rust-lang/rust/issues/35516) 🔒 @creativcoder 
- [x] [E0262](https://github.com/rust-lang/rust/issues/35517) 🔒 @creativcoder 
- [x] [E0263](https://github.com/rust-lang/rust/issues/35518) 🔒 @Limeth
- [x] [E0265](https://github.com/rust-lang/rust/issues/35309) 🔒 @mikhail-m1
- [x] [E0267](https://github.com/rust-lang/rust/issues/35519) 🔒 @shyaamsundhar
- [x] [E0268](https://github.com/rust-lang/rust/issues/35520) 🔒 @shyaamsundhar
- [x] [E0277](https://github.com/rust-lang/rust/issues/35311) 🔒 @0xmohit
- [x] [E0282](https://github.com/rust-lang/rust/issues/35312) 🔒 @medzin
- [x] [E0297](https://github.com/rust-lang/rust/issues/35521) 🔒 @arielb1
- [x] [E0301](https://github.com/rust-lang/rust/issues/35522) 🔒 @garekkream
- [x] [E0302](https://github.com/rust-lang/rust/issues/35523) 🔒 @garekkream
- [x] [E0306](https://github.com/rust-lang/rust/issues/35315) 🔒 @razielgn
- [x] [E0318](https://github.com/rust-lang/rust/issues/35322) 🔒 @acrrd
- [x] [E0322](https://github.com/rust-lang/rust/issues/35323) 🔒 @yossi-k
- [x] [E0323](https://github.com/rust-lang/rust/issues/35325) 🔒 @Keats
- [x] [E0324](https://github.com/rust-lang/rust/issues/35327) 🔒 @Keats
- [x] [E0325](https://github.com/rust-lang/rust/issues/35329) 🔒 @Keats
- [x] [E0326](https://github.com/rust-lang/rust/issues/35335) 🔒 @trixnz
- [x] [E0365](https://github.com/rust-lang/rust/issues/35633) 🔒 @mgoszcz2 
- [x] [E0368](https://github.com/rust-lang/rust/issues/35336) 🔒 @birryree
- [x] [E0370](https://github.com/rust-lang/rust/issues/35634) 🔒 @terrynsun
- [x] [E0373](https://github.com/rust-lang/rust/issues/35337) 🔒 @trixnz
- [x] [E0375](https://github.com/rust-lang/rust/issues/35635) 🔒 @mikhail-m1
- [x] [E0379](https://github.com/rust-lang/rust/issues/35338) 🔒 @kc1212
- [x] [E0384](https://github.com/rust-lang/rust/issues/35184) 🔒 @theypsilon
- [x] [E0388](https://github.com/rust-lang/rust/issues/35339) 🔒 @medzin
- [x] [E0389](https://github.com/rust-lang/rust/issues/35630) 🔒 @crypto-universe
- [x] [E0391](https://github.com/rust-lang/rust/issues/35341) 🔒 @Archytaus
- [x] [E0392](https://github.com/rust-lang/rust/issues/35631) 🔒 @canaltinova 
- [x] [E0393](https://github.com/rust-lang/rust/issues/35632) 🔒 @zjhmale
- [x] [E0394](https://github.com/rust-lang/rust/issues/35692) 🔒 @knight42
- [x] [E0395](https://github.com/rust-lang/rust/issues/35693) 🔒 @clementmiao
- [x] [E0396](https://github.com/rust-lang/rust/issues/35694) 🔒 @clementmiao
- [x] [E0403](https://github.com/rust-lang/rust/issues/35696) 🔒 @circuitfox
- [x] [E0404](https://github.com/rust-lang/rust/issues/35342) 🔒 @Archytaus
- [x] [E0407](https://github.com/rust-lang/rust/issues/35697) 🔒 @crypto-universe 
- [x] [E0408](https://github.com/rust-lang/rust/issues/35698) 
- [x] [E0409](https://github.com/rust-lang/rust/issues/35699) 🔒 @mikhail-m1
- [x] [E0422](https://github.com/rust-lang/rust/issues/35700) 🔒 @knight42
- [x] [E0423](https://github.com/rust-lang/rust/issues/35796) 🔒 @knight42
- [x] [E0424](https://github.com/rust-lang/rust/issues/35797) 🔒 @kyrias
- [x] [E0425](https://github.com/rust-lang/rust/issues/35343) 
- [x] [E0426](https://github.com/rust-lang/rust/issues/35799) 🔒 @crypto-universe
- [x] [E0428](https://github.com/rust-lang/rust/issues/35798) 🔒 @trixnz
- [x] [E0432](https://github.com/rust-lang/rust/issues/35344) 🔒 @knight42
- [x] [E0433](https://github.com/rust-lang/rust/issues/35345) 🔒 @yossi-k
- [x] [E0435](https://github.com/rust-lang/rust/issues/35801) 🔒 @shyaamsundhar
- [x] [E0437](https://github.com/rust-lang/rust/issues/35802) 🔒 @shyaamsundhar
- [x] [E0438](https://github.com/rust-lang/rust/issues/35803) 🔒 @shyaamsundhar
- [x] [E0445](https://github.com/rust-lang/rust/issues/35922) 🔒 @0xmohit
- [x] [E0446](https://github.com/rust-lang/rust/issues/35923) 
- [x] [E0449](https://github.com/rust-lang/rust/issues/35924) 
- [x] [E0450](https://github.com/rust-lang/rust/issues/35925) 🔒 @mikhail-m1
- [x] [E0451](https://github.com/rust-lang/rust/issues/35928) 🔒 @mikhail-m1
- [x] [E0453](https://github.com/rust-lang/rust/issues/35929) 🔒 @0xmohit
- [x] [E0454](https://github.com/rust-lang/rust/issues/35930) 🔒 @0xmohit
- [x] [E0458](https://github.com/rust-lang/rust/issues/35932) 🔒 @gavinb
- [x] [E0459](https://github.com/rust-lang/rust/issues/35933) 🔒 @gavinb
- [x] [E0463](https://github.com/rust-lang/rust/issues/35934) 🔒 @paulfanelli
- [x] [E0493](https://github.com/rust-lang/rust/issues/35999) 🔒 @razielgn
- [x] [E0496](https://github.com/rust-lang/rust/issues/36000) 🔒 @acrrd
- [x] [E0503](https://github.com/rust-lang/rust/issues/35703) 🔒 @wesleywiser
- [x] [E0512](https://github.com/rust-lang/rust/issues/36107) 🔒 @alygin
- [x] [E0513](https://github.com/rust-lang/rust/issues/36720) 🔒 @GuillaumeGomez
- [x] [E0516](https://github.com/rust-lang/rust/issues/36108) 🔒 @gavinb
- [x] [E0517](https://github.com/rust-lang/rust/issues/36109) 🔒 @gavinb
- [x] [E0518](https://github.com/rust-lang/rust/issues/36111) 🔒 @gavinb
- [x] [E0520](https://github.com/rust-lang/rust/issues/36112) 🔒 @0xmohit
- [x] [E0527](https://github.com/rust-lang/rust/issues/36113) 🔒 @Cobrand
- [x] [E0528](https://github.com/rust-lang/rust/issues/36194) 🔒 @EugeneGonzalez
- [x] [E0529](https://github.com/rust-lang/rust/issues/36195) 🔒 @EugeneGonzalez
- [x] [E0558](https://github.com/rust-lang/rust/issues/36196) 🔒 @abhiQmar 
- [x] [E0559](https://github.com/rust-lang/rust/issues/36197) 🔒 @Cobrand
- [x] [E0560](https://github.com/rust-lang/rust/issues/36199) 🔒 @aclarry

**Bonuses:**

Note: bonuses may require more work in the compiler and may require some exploration.
- [x] [E0046](https://github.com/rust-lang/rust/pull/36371#issuecomment-246692698)
- [x] [E0053](https://github.com/rust-lang/rust/issues/35212) 🔒 @KiChjang
- [x] [E0057](https://github.com/rust-lang/rust/issues/35214) 🔒 @acrrd 
- [x] [E0060 and E0061](https://github.com/rust-lang/rust/issues/35290) 🔒 @GuillaumeGomez
- [x] [E0072](https://github.com/rust-lang/rust/issues/35965) 🔒 @phungleson
- [x] [E0094](https://github.com/rust-lang/rust/issues/35966) 🔒 @kyrias
- [x] [E0221](https://github.com/rust-lang/rust/issues/35970) 🔒 @yossi-k
- [x] [E0537](https://github.com/rust-lang/rust/issues/36182) 🔒 @mikhail-m1
- [x] [EXXXX - Field unknown](https://github.com/rust-lang/rust/issues/36164)
- [x] [EXXXX - Fix temporary message](https://github.com/rust-lang/rust/issues/36279)
- [x] [EXXXX - Doesn't live long enough](https://github.com/rust-lang/rust/issues/36537)
- [x] [EXXXX - Better mutability](https://github.com/rust-lang/rust/issues/36412)
- [x] [E0282 - naming type variables](https://github.com/rust-lang/rust/issues/36554#issuecomment-258427960)
cc @jonathandturner 

EDIT: Don't forget to say "Part of #35233." in your PR so we can see who did what in here. Also, add "r? @GuillaumeGomez" or "r? @jonathandturner" to set us as reviewer.

**Comment by cramertj** [comment]
I guess I'll go first! I'll take 1, 4, 5, and 9.

**Comment by pcn** [comment]
I have no idea what I'm doing.  I'll take 10, 17, 23 and 25.

**Comment by silenuss** [comment]
I've also got no idea what I'm doing. I'll take 27, 29, 30, and 34. This should be fun.

**Comment by jonathandturner** [comment]
For the "I have no idea what I'm doing" crowd - I'm working on a blog post that's going up soon (hopefully tomorrow!)  It goes through all the steps of how to update the errors.

Thanks for jumping on them :)  Once the blog goes up, I'll reply here with the link.

**Comment by silenuss** [comment]
@jonathandturner Wow, that's really great of you! I kinda just assumed we were expected to figure this stuff out ourselves. The amount of effort put into this community is really awe-inspiring. Thank you so much, I'm really glad I'm not on my own on this!

**Comment by Vassah** [comment]
I've never done this before. I'd like to grab 91, 92, and 93 if that'd be helpful!

**Comment by neilzxu** [comment]
I'll grab 35, 36. Seems like digestible way to dive into open source

**Comment by shri3k** [comment]
I'll take 40, 46, 49 and 50 even though I don't know much about traits.

**Comment by jaredwy** [comment]
I will take 63 and 69.

**Comment by razielgn** [comment]
Booking 71, 76 and 79. :smiley:

**Comment by poveda-ruiz** [comment]
I'll take 81, 87, 88 and 89

**Comment by nickmass** [comment]
I will take 55, 57, and 60

**Comment by theypsilon** [comment]
I take 384 (I think there are no more available)

**Comment by lukehinds** [comment]
Might I suggest folks just take on one or two issues,  that way others wanting to learn, will have a chance to get involved too.

**Comment by cramertj** [comment]
@lukehinds Yeah, I didn't realize the precedent I was setting, nor how much interest there would be in the issue. If others who haven't contributed before would like to take any of the ones I said I would do (1, 4, 5, 9), they're welcome to it.

**Comment by TheZoq2** [comment]
I'll gladly take 4 then

**Comment by Roybie** [comment]
I'll wouldn't mind taking number 1 off you then if that's ok?

**Comment by Vassah** [comment]
@lukehinds I'll release 93 then. I can always pick it back up later if no one ends up wanting it.

**Comment by DarkEld3r** [comment]
I would like to try 93 then. Thank you.

**Comment by DevShep** [comment]
@cramertj I'll take 9 of you don't mind.

**Comment by pythoneer** [comment]
anything left? i lost count :D

**Comment by TheZoq2** [comment]
@pythoneer It looks like 5 is still available from the ones @cramertj released

**Comment by pythoneer** [comment]
i'll jump on 5 then, thx @TheZoq2

**Comment by yossi-k** [comment]
Anything else left?

**Comment by lukehinds** [comment]
@pcn @silenuss could you guys free up some of your issues for @yossi-k (and likely some others too!)

**Comment by razielgn** [comment]
Releasing 76 and 79. 😄

**Comment by GuillaumeGomez** [comment]
Wo. Much more success than expected! I intended to do a big part of it but I guess I'll just review.

**Comment by GuillaumeGomez** [comment]
@yossi-k: 76 and 79 are free.

**Comment by MuffinPimp** [comment]
I would like to try 76.

**Comment by yossi-k** [comment]
I'm on 79 then

**Comment by nindalf** [comment]
@GuillaumeGomez any left?

**Comment by faebser** [comment]
@GuillaumeGomez any left for me?

**Comment by shantanuraj** [comment]
I'll take 80

**Comment by shri3k** [comment]
@faebser @nindalf I just removed 49 & 50 from my plate if you're interested on them.

**Comment by GuillaumeGomez** [comment]
@nindalf: You can take 49 and @faebser will take 50 (thanks @shri3k!).

**Comment by nindalf** [comment]
@shri3k thanks!

**Comment by paulfanelli** [comment]
Any left to fix?

**Comment by birryree** [comment]
I'm a little late to the party but I was also wondering if there were any left to fix?

**Comment by GuillaumeGomez** [comment]
@paulfanelli, @birryree: It doesn't seem, no. Sorry. We'll maybe add more in the future but nothing sure.

**Comment by GuillaumeGomez** [comment]
@nickmass: Thanks!

@paulfanelli: You take 57 and @birryree you take 60?

**Comment by paulfanelli** [comment]
Sounds good!  I'll take 57. Thx.

**Comment by pcn** [comment]
Who still wants a ticket?  I can pass on a couple.

**Comment by bstrie** [comment]
I'd like to remind folks to also leave comments on the individual issues themselves, just so that we're totally clear about who's intending to claim what. :)

**Comment by jonathandturner** [comment]
Blog is posted!  You can read how to make the changes [here](http://www.jonathanturner.org/2016/08/helping-out-with-rust-errors.html)

If you have any questions, feel free to reach out to me or ask on #rust-internals on irc.mozilla.org (I'm jntrnr on there)

Just to second @bstrie's idea, please leave a note on the individual issues if you are taking them.

Also, this is just the first batch! So stay tuned as we fill out additional errors that need to be updated.

**Comment by jonathandturner** [comment]
Also a **huge thanks** for everyone jumping in and offering to help!

**Comment by jonathandturner** [comment]
Also, if you finish an issue, if it has a "Bonus" part and you're not able to complete it...

Once you finish, go ahead and close the issue and open a new issue for just the Bonus part.  That way, someone else can jump in and work on the bonus part.  You can cc me on the new issue and we can add it to the list above.

**Comment by jonathandturner** [comment]
For those still looking for error codes to pick up, I just added a new batch to the list on this meta-issue.

**Comment by terrynsun** [comment]
I'll take 116!

**Comment by GuillaumeGomez** [comment]
I take 132.

**Comment by Limeth** [comment]
I'll try to tackle 131.

**Comment by sciyoshi** [comment]
I can take E0124.

**Comment by intrepion** [comment]
I'd like to try 121

**Comment by Tiwalun** [comment]
I'll take 106.

**Comment by birryree** [comment]
@GuillaumeGomez I'll take 60, thanks!

**Comment by circuitfox** [comment]
I'll take 109.

**Comment by jonathandturner** [comment]
@faebser - do you want 50?

**Comment by franleplant** [comment]
Hi! I'd like to take 101 and 102

**Comment by circuitfox** [comment]
I'm also up for taking 110 and 119, if possible.

**Comment by neilzxu** [comment]
If someone wants to take 34, I accidentally took when trying to take 35/36 :D

**Comment by Roybie** [comment]
I'll take 166

**Comment by Detegr** [comment]
I'll grab 117 and 118.

**Comment by PFischbeck** [comment]
I am taking 164 and 165.

**Comment by dmjb** [comment]
I'll take 101

**Comment by prabodhtiwari** [comment]
I'll try 50

**Comment by seblw** [comment]
I would take 194.

**Comment by garekkream** [comment]
I can take 162.

**Comment by Keats** [comment]
I'll take 120

**Comment by birryree** [comment]
I'll also take E0194.

**Comment by GuillaumeGomez** [comment]
Forgot to mention this but to use the new error format, you have to run rustc like this:

```bash
RUST_NEW_ERROR_FORMAT=1 rustc file.rs
```

**Comment by GuillaumeGomez** [comment]
New errors will be added once #35274 is merged.

**Comment by yossi-k** [comment]
I'd like to take 185 as well

**Comment by medzin** [comment]
I'll take 178.

**Comment by Roybie** [comment]
taking 172

**Comment by jonathandturner** [comment]
@franleplant and @dmjb - you both called out 101, but there are others free if one of you wants to take 101 and one take another free one

**Comment by oijazsh** [comment]
I'd like to take E0107.

**Comment by jonathandturner** [comment]
I've updated the list.  There are a few free ones.  Hoping to add more soon.

**Comment by trixnz** [comment]
I can take E0062 and E0130.

**Comment by Keats** [comment]
I'll take 137 as well if no one wants it

**Comment by jonathandturner** [comment]
@keats - go ahead and take it.  I'm adding new error codes as we speak

**Comment by intrepion** [comment]
thank you @birryree for showing how to get the span labels to show up!

```sh
RUST_NEW_ERROR_FORMAT=true ./build/PLATFORM-HERE/stage1/bin/rustc src/test/compile-fail/E0121.rs
```

**Comment by jonathandturner** [comment]
For anyone interested in doing one of these, I've put a new batch of error codes into the list.

I've also updated the blog as people had issues here or there, so please check it out.

**Comment by yossi-k** [comment]
I'd like to take 322 and 433 as well.

**Comment by KiChjang** [comment]
I'll get 206.

**Comment by kc1212** [comment]
Can I take 388 please

**Comment by mikhail-m1** [comment]
I'll take 201.

**Comment by Archytaus** [comment]
I'll take E0404 and E0391 thanks.

**Comment by medzin** [comment]
I'll take 432.

**Comment by shantanuraj** [comment]
I'll try 206 and 207

**Comment by prabodhtiwari** [comment]
I'll take 432 and 425

**Comment by balrampandey19** [comment]
I'll Take E0277

**Comment by poveda-ruiz** [comment]
I'm gonna release E0088 and E0089 if someone else wants them :D

**Comment by medzin** [comment]
@prabodhtiwari ok then take the 432, I'll will take E0388

**Comment by prabodhtiwari** [comment]
@medzin sorry didnt notice you take 432 i will take another one

**Comment by medzin** [comment]
No problem :)

**Comment by mrabault** [comment]
I'll try E201.

**Comment by medzin** [comment]
I'll do E0252.

**Comment by trixnz** [comment]
Taking E0326 and E0373.

**Comment by parantapa** [comment]
Taking E0318

**Comment by Keats** [comment]
I'll take 323, 324 and 325, they seem related

**Comment by yossi-k** [comment]
i want to grab 265 as well

**Comment by mrabault** [comment]
Okay so I'll do E0229 instead. I don't understand, E0201 was free when I wrote my message ...

**Comment by jonathandturner** [comment]
@mrabault - sorry about that.  I've been updating the issue by hand when people comment on this thread.  It should be up to date now.

**Comment by mrabault** [comment]
No problem !

**Comment by razielgn** [comment]
Taking 306.

**Comment by prabodhtiwari** [comment]
i will take 0088 and 0089

**Comment by likandia** [comment]
Looks like I'm really late, but just putting this here in case someone wants to free something up :)

**Comment by GuillaumeGomez** [comment]
@likandia: E0204 and E0205 are both available.

**Comment by jonathandturner** [comment]
@likandia - I added a few more, so there should be more free

**Comment by Keats** [comment]
I'll take 243 and 244

**Comment by jonathandturner** [comment]
Helpful tip if you're working on one of these and get a little stuck: look through the PRs for error codes that have been accepted and see how they did the fix.  This might give you ideas for yours.

**Comment by munyari** [comment]
I'll take 204, 205 and 214.

**Comment by chamoysvoice** [comment]
I'll take 220, 221 :+1:

**Comment by jonathandturner** [comment]
First batch has landed!  Great work, everyone.

**Comment by KiChjang** [comment]
Taking  E0223 as well.

**Comment by seblw** [comment]
anyway wants E0191? I won't manage to find time for this.

**Comment by jonathandturner** [comment]
@seblw - I'll free up E0191 so someone else can get it.

**Comment by sciyoshi** [comment]
I'll take 191 and 225 as well

**Comment by jonathandturner** [comment]
Alright, batch #2 is in!  Another big thanks to everyone who contributed.

I've updated the signups and we're back to no more free ones.  But no have no worry, there should be more coming in the next few days.

If you find yourself getting stuck, you can ask for help from some of the other people who have gotten their PRs in.  If you don't think you'll have the time to do it, let me know and I'll free it up for someone else to take.

**Comment by sciyoshi** [comment]
@jonathandturner looks like @munyari already took E0191, but I'll do 225

**Comment by sciyoshi** [comment]
And @razielgn took 225 right before I was gonna submit :)

**Comment by crypto-universe** [comment]
Are there any free issues to fix? I'd like to jump in :)

**Comment by chamoysvoice** [comment]
@crypto-universe would you like to take 0221?

**Comment by crypto-universe** [comment]
@chamoysvoice sure! Thanks!

**Comment by jonathandturner** [comment]
Batch 3 is in!

Thanks to everyone who has been sharing their tasks.  Love seeing that.  I should have more tasks on the list starting tomorrow.

**Comment by jonathandturner** [comment]
Just a heads up - looks like the build servers are having some issues.  We're looking into it.  I'll post here when they're back up.

**Comment by likandia** [comment]
Okay if I take E0002?

**Comment by jonathandturner** [comment]
I'll add you to E0002, @likandia

**Comment by srdja** [comment]
Can I take E0008?

**Comment by jonathandturner** [comment]
@srdja - you got it

**Comment by razielgn** [comment]
Grabbing `E0026`. 😄

**Comment by munyari** [comment]
33 and 38 please!

**Comment by jonathandturner** [comment]
@razielgn and @munyari - done!

**Comment by KiChjang** [comment]
I'm taking the E0051 bonus (should be E0053 btw).

**Comment by jonathandturner** [comment]
@KiChjang - good catch :)  It's yours

**Comment by intrepion** [comment]
I noticed that some of the pull requests aren't including updated unit tests.  I'm wondering if that might count as another bonus part.

**Comment by jonathandturner** [comment]
@intrepion - we're trying to catch PRs without unit tests so that they always have one.  That said, people can always worth together.

**Comment by mikhail-m1** [comment]
I'll take 45

**Comment by jonathandturner** [comment]
Good news!  Looks like the build bots are working again, so we're back in business.

I've celebrated by adding more error codes to the list.  If you've been wanting to do a "bonus", there's a lot of new bonuses as well.

**Comment by garekkream** [comment]
I can take 301 and 302.

**Comment by arielb1** [comment]
I did the check_match ones

**Comment by lukehinds** [comment]
Hi, I think I will have a go and help my feet get wet, so  E0253 & E0254

**Comment by Limeth** [comment]
I will try to do [E0263](https://github.com/rust-lang/rust/issues/35518).

**Comment by jonathandturner** [comment]
@arielb1 - other people have signed up for some of those error codes in that PR, but there are free error codes you can take if you like.  You can also collaborate with the people who volunteered for those codes.

**Comment by circuitfox** [comment]
I'd like to take 072 and 128

**Comment by EugeneGonzalez** [comment]
I'd like to claim E0259 and E0260.

**Comment by shyaamsundhar** [comment]
I will take E0267 and E0268

**Comment by creativcoder** [comment]
I will take E0262, E0261, E0195.

**Comment by wdv4758h** [comment]
I will take E0133 and E0138.

**Comment by adrianbrink** [comment]
E0133 and E0138 please :-)

**Comment by jonathandturner** [comment]
@adrianbrink - sorry, looks like @wdv4758h beat you to it.  Maybe you two can split them and do one a piece?

I'll try to add more soon.

**Comment by wdv4758h** [comment]
Oops ! I didn't notice the message.
I have sent the PRs for both issues, but there is a bonus part in E0138.
Hopefully, I have only implemented half of the bonus part (and thinking how to solve the remaining part).
@adrianbrink I can leave the remaining part for you, if you want.

**Comment by crypto-universe** [comment]
Are there any free tasks?

**Comment by jonathandturner** [comment]
@crypto-universe sadly, not yet.  I'm hoping to put up more tasks soon.

**Comment by crypto-universe** [comment]
@jonathandturner If you have any - assign to me, please. Someone commited my task earlier, than me and now I have nothing to do :)

**Comment by lukehinds** [comment]
@crypto-universe you can take the other one I was going to work on E0254 https://github.com/rust-lang/rust/issues/35513 (I already have made a PR for E0253)

**Comment by crypto-universe** [comment]
@lukehinds Thanks a lot! I'll take E0254.

By the way, did you see such problem during check-cfail?
```
   Compiling compiletest v0.0.0 (file:///home/cryptouniverse/GIT/rust/src/tools/compiletest)
error[E0514]: found crate `log` compiled by an incompatible version of rustc
  --> src/tools/compiletest/src/main.rs:27:1
```

**Comment by jonathandturner** [comment]
@crypto-universe - I've seen that sometimes.  You'll likely need to run:

```
python src/bootstrap/bootstrap.py --clean
```

And try again

**Comment by adrianbrink** [comment]
@wdv4758h thank you :-)

**Comment by jonathandturner** [comment]
One more batch just landed.  We're one the home stretch now!

**Comment by gavinb** [comment]
@GuillaumeGomez @jonathandturner I'm also interested in picking up some tasks if there's more on the way.

**Comment by crypto-universe** [comment]
@jonathandturner do you update the task list above? I don't see free tasks.

**Comment by faebser** [comment]
Are there any left over?

**Comment by jonathandturner** [comment]
@crypto-universe and @faebser - not yet.  Hopefully soon.  We're trying to land the next set of unit tests now.

**Comment by jonathandturner** [comment]
@crypto-universe and @faebser  and others looking for more error codes, I just added a couple more.

For those who volunteered earlier but haven't finished your bugs, yet... if you're still working, awesome.  If you don't think you're going to be able to get to them, please let someone else work on them.

We're hoping to do more of these in the future, so there should be plenty of things to do :)

**Comment by mikhail-m1** [comment]
I'll take E0375

**Comment by mgoszcz2** [comment]
I can take E0365

**Comment by terrynsun** [comment]
I'll take E0380.

**Comment by canaltinova** [comment]
I'd like to take E0392

**Comment by creativcoder** [comment]
I'll take E0393

**Comment by faebser** [comment]
I take E0365

**Comment by faebser** [comment]
I'm going to work E0392

**Comment by canaltinova** [comment]
@faebser Sorry but I already claimed and working on it.

**Comment by crypto-universe** [comment]
Give me 0389, please.

**Comment by jonathandturner** [comment]
Get them while they're hot!  More errors have been added.

And we're *so close* to having merged a 100 error code updates.  Great work everyone!

**Comment by crypto-universe** [comment]
@jonathandturner I'd like to work on E0407.

**Comment by jonathandturner** [comment]
@crypto-universe and @clementmiao - I've updated the list with the ones you've chosen

**Comment by circuitfox** [comment]
I'd like to do 403.

**Comment by mikhail-m1** [comment]
I'll take 409

**Comment by faebser** [comment]
I take 409

**Comment by jonathandturner** [comment]
@faebser - @mikhail-m1 already chose 409, but it looks like you meant 408?  Signing you up for that one :)

**Comment by knight42** [comment]
I will take 394 and 422

**Comment by jonathandturner** [comment]
We have now fixed over 100 unit tests!!  🎉🎉🎉

![screen shot 2016-08-17 at 3 57 43 pm](https://cloud.githubusercontent.com/assets/547158/17756219/6d4dda26-6493-11e6-97e2-455c7aa71f47.png)

**Comment by pliniker** [comment]
I'll take a stab at 84

**Comment by mlayne** [comment]
I'll take 232.

**Comment by jonathandturner** [comment]
I've added a few more, for those interested in grabbing some.

**Comment by medzin** [comment]
I'm releasing E0432. I don't have a lot of free time recently.

**Comment by crypto-universe** [comment]
@jonathandturner Give me E0426, please.

**Comment by knight42** [comment]
I will take 54, 432 and 423

**Comment by shyaamsundhar** [comment]
I will take E0435, E0437 and E0438.

**Comment by kyrias** [comment]
I'm snatching up E0424

**Comment by trixnz** [comment]
I'll grab E0428

**Comment by nightlark** [comment]
I've never done this before, would love to jump in and try something new if there are any available.

**Comment by jonathandturner** [comment]
Would anyone with more than one error code be willing to share with @nightlark?

**Comment by mkfifo** [comment]
:+1: 
bumping to sub myself.

**Comment by 0xmohit** [comment]
I can pick up some in case someone is willing to release the unfinished ones.

**Comment by GuillaumeGomez** [comment]
@nightlark & @mkfifo & @0xmohit: I'll certainly add new err code tests today. Once merged, you'll be able to take some if you want.

**Comment by kyrias** [comment]
@mkfifo Use the subscribe button on the right sidebar instead of leaving a no-content comment.

**Comment by jonathandturner** [comment]
While we wait, would anyone who has volunteered for multiple error codes be willing to share?

**Comment by Coder206** [comment]
@GuillaumeGomez Is there any issues available to work on?

**Comment by jonathandturner** [comment]
@nightlark & @mkfifo & @0xmohit and @Coder206 - there are new ones posted

**Comment by mikhail-m1** [comment]
I'll  take E0450

**Comment by Coder206** [comment]
@jonathandturner I'd like to take E0446 as my first contribution to Rust.

**Comment by jonathandturner** [comment]
Also, heads up to people who volunteered but haven't responded after volunteering.  I'll going through and opening up some of the previous error codes to make them available for other people to work on.  If you have one that you've had for a while and have not gotten to it, you need to respond here to keep it otherwise it will be released.

**Comment by pcn** [comment]
I'm still working on E0025, it actually seems like it is making me understand the lifetimes/borrow rules.

**Comment by 0xmohit** [comment]
I'll take E0445, E0454, E0458, E0459.

**Comment by jonathandturner** [comment]
@0xmohit - can you take 1 or 2 and save some for other people?

**Comment by jonathandturner** [comment]
@pcn - thanks, I'll keep that one on you

**Comment by jonathandturner** [comment]
@0xmohit - that is, unless they're all related and easier to do in one shot

**Comment by 0xmohit** [comment]
@jonathandturner Sure. Striked out a couple.

**Comment by jonathandturner** [comment]
@0xmohit - okay you get E0445 and E0454

**Comment by nightlark** [comment]
Thanks for the new ones @jonathandturner. I'll take E0449.

**Comment by mikhail-m1** [comment]
please. assign E0451 to me

**Comment by gavinb** [comment]
@jonathandturner I'll take E0458 and E0459!

**Comment by jonathandturner** [comment]
I've released some of the older error codes that weren't being worked on.  So there are now more free ones to sign up for.

**Comment by 0xmohit** [comment]
@jonathandturner I'll take E0260, E0277 and E0453.

**Comment by GuillaumeGomez** [comment]
I take E0038 bonus.

**Comment by jonathandturner** [comment]
For those looking for bonuses, I've added a few more bonuses at the bottom of the list.

**Comment by paulfanelli** [comment]
If it's ok, I'll take E0463.

**Comment by yossi-k** [comment]
I wanna grab E0088, E0318 and E0221 (with bonuses). 
(I will release them asap if I end up not having time for them.)

**Comment by GuillaumeGomez** [comment]
@yossi-k: Then start with just one or two.

**Comment by yossi-k** [comment]
You're right. I'll take up just E0088 and E0221.

**Comment by crypto-universe** [comment]
@yossi-k E0221 is already fixed, isn't it?

**Comment by yossi-k** [comment]
@crypto-universe it's in the bonus list below

**Comment by yati-sagade** [comment]
I'll take E0049 and E0050 if that's okay

**Comment by jonathandturner** [comment]
@yati-sagade, @yossi-k, @paulfanelli - I've updated the list with your picks

**Comment by abhiQmar** [comment]
Can I take E0076 ?

**Comment by jonathandturner** [comment]
@abhiQmar - yup, E0076 is yours now

**Comment by mikhail-m1** [comment]
I'll take E0265

**Comment by jonathandturner** [comment]
@mikhail-m1 - done :)

**Comment by acrrd** [comment]
I'll take E0496

**Comment by jonathandturner** [comment]
@Coder206 Look at the list at the top ^.  Anything that doesn't have a lock and a name is available

**Comment by jonathandturner** [comment]
@accrd - done!

**Comment by razielgn** [comment]
Taking E0493.

**Comment by gavinb** [comment]
@jonathandturner I'll take E0164, E0165 and E0184

**Comment by jonathandturner** [comment]
@razielgn and @gavinb - updated!

**Comment by zjhmale** [comment]
@jonathandturner i'll take E0089, E0318 and E0393

**Comment by acrrd** [comment]
I'll take E0318

**Comment by acrrd** [comment]
@zjhmale sorry I posted without refreshing the page, I have almost finish it with only the test to fix; if is ok for you I will send the PR otherwise I move to something else

**Comment by jonathandturner** [comment]
@acrrd - since there aren't many left, what if you take E0318 and @zjhmale takes E0089 and E0393?

**Comment by acrrd** [comment]
@jonathandturner  it's ok for me

**Comment by zjhmale** [comment]
@acrrd cool, just go ahead 😄 
>@zjhmale sorry I posted without refreshing the page, I have almost finish it with only the test to fix; if is ok for you I will send the PR otherwise I move to something else

**Comment by zjhmale** [comment]
@jonathandturner ok for me
>@acrrd - since there aren't many left, what if you take E0318 and @zjhmale takes E0089 and E0393?

**Comment by gavinb** [comment]
@jonathandturner I've finished E0164, E0165 and E0184. They're each in their own commit in my fork, but because I messed up resyncing my local and git repos, so I can't make a PR for the newer changes yet. Once my first PR is closed and merged, I'll update my tree to get back in sync with rust-lang and then shoot off a PR for the next 3.

Happy to do some more if any become available. Might take a look at some of the bonus items too.

**Comment by GuillaumeGomez** [comment]
@gavinb: You can always create a new branch, put your commits in it (or re-doing changes by hand and commit them) and then force push it on your branch.

**Comment by jonathandturner** [comment]
I just added a few more to the list if people are looking for errors to update.

**Comment by mrstork** [comment]
I'll take on E0512

**Comment by jonathandturner** [comment]
@mrstork - done!

**Comment by ashrko619** [comment]
Can I take E0527?

**Comment by gavinb** [comment]
@jonathandturner  I'll take E0516, E0517, E0518. And I cheekily just put up a PR for these. Hopefully nobody else picked them up in the meantime!

**Comment by 0xmohit** [comment]
@jonathandturner I'll take E0520.

**Comment by phungleson** [comment]
I can take a look at E0516 E0517 E0518 E0520 E0527

**Comment by jonathandturner** [comment]
@ashrko619 @gavinb @0xmohit - updated!

@phungleson - looks like those are taken, but hopefully we'll have a few more soon

**Comment by KiChjang** [comment]
Just so everyone knows, E0057 and E0072 are still open, and require assistance.

**Comment by GuillaumeGomez** [comment]
@phungleson: You can look at those proposed by @KiChjang.

**Comment by phungleson** [comment]
@GuillaumeGomez I can look into E0072 first.

**Comment by GuillaumeGomez** [comment]
@phungleson: Sure.

**Comment by jonathandturner** [comment]
@GuillaumeGomez @KiChjang @phungleson - isn't E0072 already finished?

**Comment by KiChjang** [comment]
@jonathandturner I haven't worked on it at all, and I don't think anyone else has claimed it, or submitted a PR for it? (I meant the bonus section btw)

**Comment by phungleson** [comment]
@jonathandturner @GuillaumeGomez @KiChjang Yes I meant the bonus for E0072

**Comment by athulappadan** [comment]
@jonathandturner Is it okay to take up those ones which are assigned but not got any response for a long time.?

**Comment by GuillaumeGomez** [comment]
@athulappadan: No. Please wait for @jonathandturner or me say that this is available again.

**Comment by acrrd** [comment]
I'll take E0057

**Comment by jonathandturner** [comment]
Alright - @phungleson - you've got bonus E0072, and @acrrd - you have bonus E0057

**Comment by Cobrand** [comment]
I'd like to take E0559 and E0560 if possible

**Comment by EugeneGonzalez** [comment]
I will take E0528 and E0259.

**Comment by mikhail-m1** [comment]
I'd like to take  E0537 bonus

**Comment by EugeneGonzalez** [comment]
@jonathandturner Yes, sorry about the mix-up.

**Comment by jonathandturner** [comment]
@EugeneGonzalez - no worries :)

@mikhail-m1 - it's yours!

**Comment by abhi-sm** [comment]
I'd like to take E0558

**Comment by abhiQmar** [comment]
I'd like to take E0558

**Comment by jonathandturner** [comment]
@abhi-sm - updated

@abhiQmar - looks like someone beat you to it

**Comment by aclarry** [comment]
@Cobrand, I started working on E0560 (didn't see that you took it), I can continue working on it if you don't mind.

**Comment by Cobrand** [comment]
> @Cobrand, I started working on E0560 (didn't see that you took it), I can continue working on it if you don't mind.

@aclarry Tbh I had started working on it too, but honestly for the amount of time that I've spent on it, this isn't an issue at all.

 @jonathandturner I leave E0560 in @aclarry 's hands

**Comment by Cobrand** [comment]
@aclarry Actually, I have to take care of E0559 too, and we'll have to have a look at exactly the same code (`src/librustc_typecheck/check/mod.rs` lines 3114-3129 ); there is some refactor to do here for both of us. Did you start working on that as well ?

One of us will have to do a merge/rebase at some point if we open our PRs at the same time; I don't mind doing it, but we would probably be better if one did his PR before the other. Since you already started working on it, could you please open your PR quickly so I can adapt to what you've changed ?

Either that, or we can do both of our commits on a common branch (in your fork or mine), and then do a single and grouped PR.

**Comment by aclarry** [comment]
@Cobrand I haven't started working on that section.  I'm going to be offline for the next 2 days, so I will put my work on E0560 on pause, and I can merge your changes from E0559.

I'm actually quite new to working on rust, so if you think E0559 and E0560 is a one or two day project for you, I'm also happy to let you go ahead with working on those together, and I can learn from the changes you made for E0560 :) (Take this as leave to take back E0560 if that's easier for you.)

**Comment by Cobrand** [comment]
@aclarry I did my PR : https://github.com/rust-lang/rust/pull/36267
I think by the time you get back in 2 days it will be merged ( I hope ), so everything's fine.

If it isn't, you can still take a look at what I've changed so it doesn't conflict.
(Btw, please rename E560.rs into E0560.rs in your PR, I almost filed an issue because E0560 wasn't here ...)

**Comment by jonathandturner** [comment]
Just a heads up that it's probably time to start letting more people work on the errors again.  Since we're down to only about 30 left, it may be time to free them up and let other people volunteer.

Also, if you're waiting on a review please either reply here or tag me (or @GuillaumeGomez) in the PR.  I've been busy the last few weeks preparing for RustConf but now I'm back.

**Comment by bstrie** [comment]
Paging @arielb1, @pcn, @neilzxu, @GuillaumeGomez, @paulfanelli, @yossi-k, @chamoysvoice, @terrynsun, @faebser, @knight42, @prabodhtiwari, @shyaamsundhar, @Coder206, @nightlark, @mrstork, @aclarry 

If I've paged you here, it means that you've claimed one of the still-open bugs above! We understand that life gets busy sometimes, so if you don't have the time to work on those bugs, let us know and we'll free them up for others. :) And if you do want to still work on those bugs but just haven't gotten around to it yet, let us know soon before we start assigning them to new people!

**Comment by knight42** [comment]
@bstrie I have finished my tasks, but it seems that the page didn't get updated.

**Comment by terrynsun** [comment]
Ooops, I accidentally closed my PR early for E0370. I can fix it by this weekend.

**Comment by bstrie** [comment]
@knight42, great, I've updated the issues, thanks. :)

@terrynsun, looking forward to it!

**Comment by GuillaumeGomez** [comment]
Mine has already a PR opened. I wait for @jonathandturner's fix to get merged to update mine.

**Comment by aclarry** [comment]
@bstrie my PR for E0560 is in the process of being merged in, the issue should be closed shortly :)

**Comment by neilzxu** [comment]
@bstrie whoops, had my fix, just need to rebase. I'll finish it up by next week.

**Comment by chamoysvoice** [comment]
@bstrie I need to rebase, I'll be up this weekend :zap:

**Comment by jonathandturner** [comment]
For people needing span merging, my PR landed.  You can find `merge_spans` in CodeMap now.

**Comment by rrichardson** [comment]
I didn't see 513 on the list but it needs some help : 

```
error[E0513]: no type for local variable 322
   --> src/gelf.rs:103:41
    |
103 |             mem::transmute_copy::<&[u8; hdr_sz], GelfChunkHeader>(&b[..hdr_sz]);
    |                                         ^^^^^^

```
hdr_sz in this case was initialized with :  std::mem::size_of::<GelfChunkHeader>()

**Comment by jonathandturner** [comment]
If someone wants to grab the error @rrichardson points out, make sure to submit a unit test along with it.

**Comment by bstrie** [comment]
I've added an issue for E0513 in the checklist.

**Comment by GuillaumeGomez** [comment]
@bstrie: Thanks!

I took it.

**Comment by jonathandturner** [comment]
Greetings all!

Like @bstrie says, it's time for one final push to finish this metabug and call it done.

To get there, we're going to try something a little different.  We've opened up the bugs so anyone can send in changes rather than having to sign up.  If you create a PR, tag me in it for the review.  Let's see how many of these remaining ones we can get done.

Let the last push begin!

**Comment by Coder206** [comment]
@jonathandturner So I can continue E0446?

**Comment by jonathandturner** [comment]
@Coder206 - yup, anyone can work on E0446.  If you're working on it now, you might want to finish it up and send in the PR before someone gets to it first.

**Comment by jonathandturner** [comment]
Bonuses!  There are also new bonuses if you want to work on a bonus

**Comment by mikhail-m1** [comment]
@jonathandturner  Any free bonuses? I'd like to take one

**Comment by jonathandturner** [comment]
Yes! to @mikhail-m1 and anyone interested, there are 5 bonuses on the bonus list that haven't been completed yet.  Please feel free to grab one!

**Comment by mikhail-m1** [comment]
@jonathandturner I'll take 'Doesn't live long enough'

**Comment by gavinb** [comment]
@jonathandturner Next time you update the list, I've completed the "EXXXX - Field unknown" bonus with https://github.com/rust-lang/rust/pull/36798. Having a look at the others now.

**Comment by jonathandturner** [comment]
@gavinb - great! It's updated

**Comment by jonathandturner** [comment]
Long time no see!  I just added a really juicy task with some good bonus material.  Niko has gone into great pains to list out the steps that need to go into this, and it's fixing a common pain point: seeing `_` instead of the name of the type variable.

You can see the steps here: https://github.com/rust-lang/rust/issues/36554#issuecomment-258427960

**Comment by mikhail-m1** [comment]
I've almost done EXXXX - Fix temporary message and EXXXX - Doesn't live long enough, working on EXXXX - Better mutability

**Comment by bstrie** [comment]
@jonathandturner Looks like all these are done except for the "bonus" bugs. Worth closing this issue?

**Comment by GuillaumeGomez** [comment]
Make a smaller one would be nice. I'll close once the new smaller one has been created.

**Comment by estebank** [comment]
@GuillaumeGomez I feel like this ticket can now be closed, as every action item has been taken care of.

**Comment by Putaek84** [comment]
Putaek84
